tests/foreman/cli/test_hostcollection.py

```diff
@@ -285,7 +282,10 @@ def test_positive_copy_by_id(module_org, module_target_sat):
 
 
 @pytest.mark.upgrade
-def test_positive_register_host_ak_with_host_collection(module_org, module_ak_with_cv, target_sat):
+@pytest.mark.rhel_ver_match('N-2')
```

Comment:
I think you don't need any specific RHEL here, so I'd suggest using default RHEL for this test

```suggestion
@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
```

Reply:
Done